core/tests/protractor.conf.js

```diff
@@ -338,6 +339,27 @@ exports.config = {
       }));
     }
 
+    // Adding a video reporter. For more information see
+    // https://www.npmjs.com/package/protractor-video-reporter
+
+    var ffmpegArguments = [
+      '-y',
+      '-r', '30',
+      '-f', 'x11grab',
+      '-s', '1366x768',
+      '-i', process.env.DISPLAY,
+      '-g', '300',
+      '-vcodec', 'qtrle',
+    ]
+
+    jasmine.getEnv().addReporter(new VideoReporter({
```

Comment:
Do we need to block generating video for developers?

Reply:
Sorry, I'm not sure what you mean?

Reply:
I mean are we going to generate video when running the e2e test locally?

Reply:
Ah, I see what you mean. I'm not sure how we're going to stop the tests running locally, though?

Reply:
@U8NWXD Do you have any suggestions on this? Should we have a flag to enable the video feature?